backend/app/helpers/spree/admin/stores_helper.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  module Admin
+    module StoresHelper
+      def selected_checkout_zone(store)
```

Comment:
@przemosk let's change this method to return the zone object, not the ID

Reply:
@damianlegawiec fixed